### PR TITLE
Fix UI Thread deadlock

### PR DESCRIPTION
**Pull request by ragner**
Fixes to UI thread deadlock and improvements to test https://github.com/Samsung/GearVRf/pull/1973

GearVRf-DCO-1.0-Signed-off-by: Ragner Magalhaes <ragner.n@samsung.com>

### CODE REVIEW

**Comment by liaxim** [comment]
Extensions/WidgetPlugin doesn't compile for me.

**Comment by liaxim** [comment]
gvr-3dcursor doesn't compile for me.

**Comment by ragner** [comment]
Hi @liaxim, I have fixed the 3dcursor issues.
You need update the framework branch too.

**Comment by liaxim** [comment]
gvr-remote-scripting doesn't compile

**Comment by ragner** [comment]
@liaxim Sorry,
everything should compile now!

File: gvr-renderableview/app/src/main/res/layout/activity_main.xml [hunk 1]
```diff
@@ -2,8 +2,9 @@
 <LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
     android:layout_width="wrap_content"
     android:layout_height="wrap_content"
+    android:background="@color/white_transparent"
```
Comment (liaxim) [inline gvr-renderableview/app/src/main/res/layout/activity_main.xml]: I get an aapt error: ``AAPT: error: resource color/white_transparent (aka org.gearvrf.renderableview:color/white_transparent) not found.``

**Comment by liaxim** [comment]
Looks good. Planning to merge on Friday.

**Pull request merged**